controller/array_action/array_mediator_interface.py

```diff
@@ -30,14 +30,14 @@ def disconnect(self):
         raise NotImplementedError
 
     @abstractmethod
-    def create_volume(self, volume_name, size_in_bytes, capabilities, pool):
+    def create_volume(self, volume_name, size_in_bytes, space_efficiency, pool):
         """
         This function should create a volume in the storage system.
 
         Args:
             volume_name      : name of the volume to be created in the stoarge system
             size_in_bytes : size in bytes of the volume
-            capabilities  : dict of capabilities {<capbility_name>:<value>}
+            space_efficiency  : space efficiency type (None for default)
```

Comment:
[again](https://github.com/IBM/ibm-block-csi-driver/pull/212#discussion_r546399417), no need for `type`

Reply:
yes, done